src/base-client-helpers_test.ts

```diff
@@ -0,0 +1,141 @@
+import { assertEquals } from "https://deno.land/std@0.185.0/testing/asserts.ts";
+import { _internals, getUserAgent } from "./base-client-helpers.ts";
+import { serializeData } from "./base-client-helpers.ts";
+import { assertSpyCalls, stub } from "./dev_deps.ts";
+
+Deno.test(_internals.getModuleVersion.name, async (t) => {
```

Comment:
Can you prefix all the test names with the module name it exists in, i.e. `base-client-helpers`? This way if the test fails in the future while we are working on something else, it is a bit more immediately obvious what failed and from where.

Reply:
I've added the `base-client-helpers` module but I've hard coded it since I could not find a way to get this  value dynamically let me know if you know a better way to do this?

Reply:
That's fine!